src/test/java/com/debatetimer/service/parliamentary/ParliamentaryServiceTest.java

```diff
@@ -0,0 +1,143 @@
+import java.util.Optional;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+class ParliamentaryServiceTest extends BaseServiceTest {
+
+    @Autowired
+    ParliamentaryService parliamentaryService;
+
+    @Nested
+    class save {
+
+        @Test
+        void 의회식_토론_테이블을_생성_한다() {
+            Member chan = fixtureGenerator.generateMember("커찬");
+            ParliamentaryTableCreateRequest chanTableRequest =  dtoGenerator.generateParliamentaryTableCreateRequest("커찬의 토론 테이블");
+            TableInfoCreateRequest requestTableInfo = chanTableRequest.info();
+            List<TimeBoxCreateRequest> requestTimeBoxes = chanTableRequest.table();
+
+            ParliamentaryTableResponse savedTableResponse = parliamentaryService.save(chanTableRequest, chan);
+            Optional<ParliamentaryTable> foundTable = parliamentaryTableRepository.findById(savedTableResponse.id());
+            List<ParliamentaryTimeBox> foundTimeBoxes = timeBoxRepository.findAllByParliamentaryTable(foundTable.get());
+
+            assertAll(
+                    () -> assertThat(foundTable.get().getName()).isEqualTo(requestTableInfo.name()),
+                    () -> assertThat(foundTimeBoxes).hasSize(requestTimeBoxes.size())
+            );
+        }
+    }
+
+    @Nested
+    class findTable {
+
+        @Test
+        void 의회식_토론_테이블을_조회_한다() {
+            Member chan = fixtureGenerator.generateMember("커찬");
+            ParliamentaryTable chanTable = fixtureGenerator.generateParliamentaryTable(chan);
+            fixtureGenerator.generateParliamentaryTimeBox(chanTable, 1);
+            fixtureGenerator.generateParliamentaryTimeBox(chanTable, 2);
+
+            ParliamentaryTableResponse foundResponse = parliamentaryService.findTable(chanTable.getId(), chan);
+
+            assertAll(
+                    () -> assertThat(foundResponse.id()).isEqualTo(chanTable.getId()),
+                    () -> assertThat(foundResponse.table()).hasSize(2)
+            );
+        }
+
+        @Test
+        void 회원_소유가_아닌_테이블_조회_시_예외_발생() {
+            Member chan = fixtureGenerator.generateMember("커찬");
+            Member coli = fixtureGenerator.generateMember("콜리");
+            ParliamentaryTable chanTable = fixtureGenerator.generateParliamentaryTable(chan);
+
+            assertThatThrownBy(() -> parliamentaryService.findTable(chanTable.getId(), coli))
+                    .isInstanceOf(DTClientErrorException.class)
+                    .hasMessage("회원님의 토론 테이블을 찾을 수 없습니다.");
+        }
+    }
+
+    @Nested
+    class updateTable {
+
+        @Test
+        void 의회식_토론_테이블을_수정_한다() {
+            Member chan = fixtureGenerator.generateMember("커찬");
+            ParliamentaryTable chanTable = fixtureGenerator.generateParliamentaryTable(chan);
+            ParliamentaryTableCreateRequest renewTableRequest =  dtoGenerator.generateParliamentaryTableCreateRequest("새로운 테이블");
```

Comment:
```suggestion
            ParliamentaryTableCreateRequest renewTableRequest = dtoGenerator.generateParliamentaryTableCreateRequest("새로운 테이블");
```
이 부분을 복붙했는지 해당 코드를 사용하는 곳에서 전부 공백이 2개 들어가있네요.
[참고](https://velog.io/@pgmjun/IntelliJ-%EC%BD%94%EB%93%9C-%EC%8A%A4%ED%83%80%EC%9D%BC%EC%9D%84-%EC%84%A4%EC%A0%95%ED%95%B4%EB%B3%B4%EC%9E%90-feat.%EC%9A%B0%ED%85%8C%EC%BD%94)

이 글 보고 파일 저장할 때 자동 정렬 하도록 해놓으면 좋을 것 같아요. 이미 작성된 파일들은 직접 자동정렬(`option + command + L`)을 해주세요.

Reply:
`Ctrl + Alt + L` 입니다 ^^

Reply:
설정되어있으나 자동정렬이 습관이 잘 안잡혀있는 것 같습니다 😭  유의하겠슴다ㅠㅠ